tools/internal_ci/linux/grpc_reverse_test_python.sh

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# To grpc repo root
+cd $(dirname $0)/../../..
+ROOT=$(pwd)
+
+source tools/internal_ci/helper_scripts/prepare_build_linux_rc
+
+python3 -m pip install -U nox virtualenv
+
+tools/run_tests/task_runner.py -f artifact manylinux2010 x64 cp38-38 -j 1
+
+ls -al ${ROOT}/artifacts
+ls -al ${ROOT}/artifacts/python_manylinux2010_x64_cp38-cp38
+ls -al ${ROOT}/artifacts/python_manylinux2010_x64_cp38-cp38/grpcio-1.29.0.dev0-cp38-cp38-manylinux2010_x86_64.whl
+
+git clone https://github.com/googleapis/python-firestore
```

Comment:
note that all these commands will run directly on the bare kokoro VM (not inside docker), which at some point might be problematic (e.g. if the kokoro image is upgraded etc).

Reply:
I will test if porting this test to docker is feasible or not. If so, it should save us a lot of effort to install all Python versions.